docs/how-tos/sensor-data-query-sdk.md

```diff
@@ -76,14 +86,17 @@ pip install viam-sdk
 To access your machines using the Python SDK, you must use an API key:
 
 ```sh {class="command-line" data-prompt="$"}
-viam organizations api-key create --org-id <org-id> --name my-api-key
+viam organizations api-key create --org-id=<org-id> --name=my-api-key
 ```
 
+This command uses the Viam CLI.
+You can use [`viam organizations list`](/cli/#organizations) to retrieve your organization's ID.
```

Comment:
Because we can now get API keys from the organization settings page, we could change this. Then there's no need to install the CLI anymore. Fewer clicks away from this page

Reply:
Are you sure? IMO its a bit simpler than the manual instructions for retrieving api key and organization ids and that would require them to click around in the app anyways. I will change if you think 100%